### ISSUE TITLE
Rename startFrom()

### ISSUE THREAD

**Issue by cyroxx** [jlolling/talend_routines#1]
I find the name of the method StringUtil.startFrom() is misleading. It suggests to "use the substring starting at index `index` until the end" (which would probably make a useful method of its own).

How about renaming it to `left()`, similar to [StringUtils.left()](http://commons.apache.org/proper/commons-lang//apidocs/org/apache/commons/lang3/StringUtils.html#left-java.lang.String-int-) in the Apache Commons library?

**Comment by jlolling** [comment]
I agree with you. I will add a method left() but will keep the old one for compatibility reason.
